assets/course-theme/blocks/blocks.js

```diff
@@ -123,6 +143,10 @@ const blocks = [
 		...meta,
 		title: __( 'Contact Teacher', 'sensei-lms' ),
 		name: 'sensei-lms/button-contact-teacher',
+		description: __(
```

Comment:
I had a Contact Teacher block in a post, and it stopped working.

<img width="962" alt="Screen Shot 2022-01-28 at 18 15 33" src="https://user-images.githubusercontent.com/876340/151622490-3969aacc-dea5-4ce5-943f-29391c8d44a1.png">
 
I imagine that the reason is that we're registering this block there with the same name, right? Maybe it shouldn't happen because your check? `if ( ! getBlockType( name ) )`? Also, not sure if we should register this block with the same name. Probably would cause some issues.

Reply:
Hm, right, the theme block gets loaded instead of the existing one. I'll try to tweak the order, or just remove the theme one.

Reply:
Fixed in fd727bf4b576a66772514d85c777d5da3b63213c to always use the original contact teacher block in the site editor too.